### PR TITLE
Cancel order

### PR DESCRIPTION
**Pull request by MarioRodrigues10**

### CODE REVIEW

File: .tool-versions [hunk 1]
```diff
@@ -1,3 +1,3 @@
-erlang 25.0.3
+erlang 25.1.2
```
Comment (RuiL1904) [inline .tool-versions:1]: I don't think this version upgrade belongs in this PR 😅

File: lib/store_web/live/order_live/show.ex [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+           socket
+           |> put_flash(:success, "Product purchased successfully!")
+          }
+        {:error, %Ecto.Changeset{} = changeset} ->
+          {:noreply, assign(socket, :changeset, changeset)}
+      end
+    end
+
+    Repo.delete!(order)
+  end
```
Comment (RuiL1904) [inline lib/store_web/live/order_live/show.ex:49]: I think I'm not understanding this function 🤔
It handles the `"draft"` event, but purchases all the products inside a order and then delete it. I'm quite confused 😅

Comment (MarioRodrigues10) [reply lib/store_web/live/order_live/show.ex:49]: Basically I put every product that was in the ```"ordered order"``` in the ```"draft order"``` , so then it makes no sense to still have the order in "ordered" so i deleted that order. :)

Comment (RuiL1904) [reply lib/store_web/live/order_live/show.ex:49]: Hmm, I see.
I don't agree with that. I think a user should only have the possibility to cancel an ordered order.

**Review by RuiL1904 — APPROVED** [review]

**Pull request merged**